.expeditor/verify.pipeline.yml

```diff
@@ -0,0 +1,143 @@
+---
+steps:
+
+#########################################################################
+  # Component Tests
+#########################################################################
+
+- label: pushy-server-schema
```

Comment:
I think we need to change this; the schema is a prep stage for the main push jobs tests and doesn't stand alone.

all we should need is to have make; make test run once the schema is setup.